tests/postgres_tests/test_array.py

```diff
@@ -173,12 +173,32 @@ def test_in(self):
             self.objs[:2]
         )
 
+    @unittest.expectedFailure
```

Comment:
Are you going to create a ticket for this issue we could reference in the comment?